src/module.ts

```diff
@@ -48,6 +49,28 @@ export default defineNuxtModule<ModuleOptions>({
     nuxt.options.alias['#cookie-control'] = runtimeDir
     nuxt.options.build.transpile.push('#cookie-control')
 
+    // Import styles file
+    nuxt.options.css.push(resolver.resolve('./runtime/styles.css'))
+    // Install tailwindcss via nuxt module
+
+    await installModule('@nuxtjs/tailwindcss', {
+      // module configuration
+      exposeConfig: true,
+      viewer: false,
+      config: {
+        theme: {
+          extend: {
+            colors: moduleOptions.colors as unknown as any,
+          },
+        },
+        darkMode: 'class',
+        content: [
+          resolver.resolve('./runtime/**/*.{vue,mjs,ts}'),
+          resolver.resolve('./runtime/*.{mjs,js,ts}'),
+        ],
+      },
```

Comment:
Does this config merge with a user's tailwind configuration in the downstream projects?

Reply:
This is one of the main things I have to still check. Not sure atm